quick_start/sample_app.py

```diff
@@ -0,0 +1,382 @@
+import enum
+import json
+import uuid
+from typing import ClassVar
+import inflection as inflection
+from azure.kusto.ingest import QueuedIngestClient
+from quick_start.utils import Utils, AuthenticationModeOptions
+from azure.kusto.data import DataFormat, KustoClient
+
+
+class SourceType(enum.Enum):
+    """
+    SourceType - represents the type of files used for ingestion
+    """
+
+    local_file_source = ("localFileSource",)
+    blob_source = ("blobSource",)
+
+
+class ConfigData:
+    """
+    ConfigData object - represents a file from which to ingest
+    """
+
+    source_type: ClassVar[SourceType]
```

Comment:
Classvar in python roughly means the same as static variables in c#. Since these are all member variables, you don't need ClassVar (and in configjson too)